Integrations/Cloaken/Cloaken.yml

```
@@ -1,41 +1,41 @@
-category: Data Enrichment & Threat Intelligence
 commonfields:
-  id: Cloaken
+  id: Cloaken_dev
   version: -1
+name: Cloaken_dev
+display: Cloaken_dev
+category: Data Enrichment & Threat Intelligence
+description: Unshorten URLs onsite using the power of a Tor proxy server to prevent
+  leaking IP addresses to adversaries.
 configuration:
 - display: Server URL (e.g., https://cloaken.cypherint.com)
   name: server_url
-  defaultvalue: "https://cloaken.cypherint.com"
-  required: true
+  defaultvalue: https://cloaken.cypherint.com
   type: 0
+  required: true
 - display: Username
   name: credentials
-  required: true
+  defaultvalue: ""
   type: 9
+  required: true
 - display: Use system proxy settings
   name: proxy
-  required: false
+  defaultvalue: ""
   type: 8
+  required: false
 - display: Trust any certificate (not secure)
   name: insecure
-  required: false
+  defaultvalue: ""
   type: 8
-description: Unshorten URLs onsite using the power of a Tor proxy server to prevent leaking IP addresses to adversaries.
-display: Cloaken
-name: Cloaken
+  required: false
 script:
+  script: ''
+  type: python
```

also add subtype here:
```suggestion
  type: python
  subtype: python3
```